src/modules/currency/wallet/currency-wallet-api.js

```diff
@@ -214,6 +214,10 @@ export function makeCurrencyWalletApi (
       return engine.makeSpend(spendInfo)
     },
 
+    async sweepPrivateKeys (spendInfo: EdgeSpendInfo): Promise<EdgeTransaction> {
```

Comment:
This needs to be added to `EdgeCurrencyWallet`, then, not just the engine.

Reply:
The `EdgeCurrencyWallet` already has this method defined in its API: `sweepPrivateKey(keyUri: string): Promise<void>`. At the very least, we should delete the old function. However, I think the API you have is just... wrong. The API we already have makes more sense.